tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -234,8 +234,14 @@ delimiter = '"'
 # Line terminator. Empty value means both "\n" (LF) and "\r\n" (CRLF) are line terminators.
 terminator = ''
 # Whether the CSV files contain a header.
-# If `header` is true, the first line will be skipped.
+# If `header` is true, TiDB Lightning treats the first row as a table header and does not import it as data. If `header` is false, the first row is also imported as CSV data.
 header = true
+# Whether the columns in the CSV file header are matched to those defined in the target table.
+# The default value is set to "true", which means that TiDB Lightning will match the column names in the CSV header with those in the target table, so that even if the order of the columns is different between the two, the data can still be imported successfully.
```

Comment:
which means that you have confirmed that the column names in the CSV header with those in the target table are consistent, so that even if the order of the columns is different between the two, the data can still be imported successfully by Lightning.